packages/ui/src/views/repo/pull-request/components/pull-request-labels-list.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, Icon } from '@components/index'
+
+interface LabelsListProps {
+  labels?: { key?: string; id?: number; color?: string }[]
+  handleDelete?: (id: number) => void
+  addLabelError?: string
+  removeLabelError?: string
+}
+
+const LabelsList: React.FC<LabelsListProps> = ({ labels, handleDelete, addLabelError, removeLabelError }) => (
```

Comment:
is this a new view being added in this PR?
is this required?

Reply:
@cjlee01 I see you added this, do we need it?

Reply:
There's a similar component here `packages/ui/src/views/repo/pull-request/components/labels/pull-request-labels-list.tsx`

Reply:
@cjlee1 I've removed this file for now, I see it's not used anywhere. Please add it back in a separate PR.

Reply:
I think I added then it was refactored but this reference was not used and that’s why I see duplication 
@vardanbansal-harness